crates/rooch-sequencer/src/actor/sequencer.rs

```diff
@@ -64,6 +62,24 @@ impl SequencerActor {
         let tx_order_signature = Signature::new_hashed(&witness_hash.0, &self.sequencer_key)
             .as_ref()
             .to_vec();
+        let tx = Self::build_ledger_transaction(tx_data, tx_order, tx_order_signature)?;
+
+        self.rooch_store.save_transaction(tx.clone())?;
+        debug!("sequencer tx: {} order: {:?}", hash, tx_order);
+        self.last_order = tx_order;
+        self.rooch_store
+            .save_sequencer_order(SequencerOrder::new(self.last_order))?;
+
+        Ok(tx)
+    }
+
+    pub fn build_ledger_transaction(
+        tx_data: LedgerTxData,
+        tx_order: u64,
+        tx_order_signature: Vec<u8>,
+    ) -> Result<LedgerTransaction> {
+        let now = SystemTime::now();
+        let tx_timestamp = now.duration_since(SystemTime::UNIX_EPOCH)?.as_millis() as u64;
```

Comment:
the genesis tx timestamp should be a fixed time, use the timestamp in GenesisConfig.